docs/content/doc/usage/reverse-proxies.en-us.md

```diff
@@ -221,6 +221,9 @@ If you wish to run Gitea with IIS. You will need to setup IIS with URL Rewrite a
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
+    <system.web>
```

Comment:
Ok so looking at the docs here it appears by default that the following characters are disallowed:

```
<,>,*,%,&,:,\
```
I guess we really only need the `:` but gitea is safe with all of these characters.